src/main/java/org/alliancegenome/curation_api/auth/AuthenticationFilter.java

```diff
@@ -41,38 +41,27 @@
 public class AuthenticationFilter implements ContainerRequestFilter {
 
 	@Inject
-	@AuthenticatedUser
-	Event<Person> userAuthenticatedEvent;
+	@AuthenticatedUser Event<Person> userAuthenticatedEvent;
 
-	@Inject
-	AuthenticationService authenticationService;
+	@Inject AuthenticationService authenticationService;
 
-	@Inject
-	PersonDAO personDAO;
+	@Inject PersonDAO personDAO;
```

Comment:
I like to have the annotation on top of the line it's annotating as it is easier to read for me. 
Can we not change this?